inference-engine/tests/functional/shared_test_classes/src/single_layer/gather_elements.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "ngraph_functions/builders.hpp"
+#include "shared_test_classes/single_layer/gather_elements.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string GatherElLayerTest::getTestCaseName(const testing::TestParamInfo<GatherElParams>& obj) {
+    InferenceEngine::SizeVector dataShape, indicesShape;
+    InferenceEngine::Precision dPrecision, iPrecision;
+    int axis;
+    std::string device;
+    std::tie(dataShape, indicesShape, axis, dPrecision, iPrecision, device) = obj.param;
+
+    std::ostringstream result;
+    result << "DS=" << CommonTestUtils::vec2str(dataShape) << "_";
+    result << "IS=" << CommonTestUtils::vec2str(indicesShape) << "_";
+    result << "Ax=" << axis << "_";
+    result << "DP=" << dPrecision.name() << "_";
+    result << "IP=" << iPrecision.name() << "_";
+    result << "device=" << device;
+
+    return result.str();
+}
+
+void GatherElLayerTest::SetUp() {
```

Comment:
Looks like "GatherElementsLayerTest" would be more readable.

Reply:
Fixed